### ISSUE TITLE
Suggestion: Action Picker on key press.

### ISSUE THREAD

**Issue by Roocket4** [InZidiuZ/op-framework-bug-tracker#72]
Something in the likes of the action bar with weapons but with actions you can perform that are now weapon related.
For instance you press or hold a key (lets say F1) and you get the options maybe like in The Division 2 emote wheel with small icons representing what it is you will do. And on that wheel there would be like carry, handcuff, dice, check both bank and wallet balance simultaneously etc..

So basically remove the need of typing the / commands all the time.

**Comment by InZidiuZ** [comment]
Hey,

This probably won't happen as the "action commands" are usually kept very short, and also often given shortcuts (/sc for /softcuff, for example).

We're also quite limited as to how many hotkeys we could actually use. Most of the available a-z keys are already used for various things, so is F1-F12 and 1-7.

If you're not a fan of more command based actions, feel free to set up custom hotkeys, using .ahk, Razer Synapse, or any other software that lets you do this.

**InZidiuZ**
*OP-FW Developer*
